db/deploy/views/applications_view.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy ccbc:views/applications_view to pg
+-- requires: tables/applications
+
+begin;
+
+CREATE OR REPLACE VIEW ccbc_public.applications_view AS SELECT * FROM ccbc_public.applications;
+
+comment on view ccbc_public.applications_view is 'View containing thelist of all respondents applications';
```

Comment:
Small typo here `View containing thelist` -> `View containing the list`

Reply:
Thanks for noticing! Fixed...